GitExtensions/app.config

```diff
@@ -15,10 +15,6 @@
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <probing privatePath="plugins" />
-      <dependentAssembly>
-        <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
-      </dependentAssembly>
```

Comment:
Any reason why this is deleted?

Reply:
https://github.com/gitextensions/gitextensions/pull/6005/commits/cfdae38a29fab95177de330a2243b648c08cf051

![image](https://user-images.githubusercontent.com/3629489/50741308-804adf80-11c9-11e9-9a8d-7069f523d8b7.png)

Reply:
Nice! 👍